docs/flare_overview.rst

```diff
@@ -4,50 +4,89 @@
 
+  * A runtime environment enabling data scientists and researchers to easily carry out FL experiments in a real-world scenario. Nvidia FLARE supports multiple task execution, maximizing data scientist's productivity. 
+  
+  * System capabilities to stand up Federated learning with high availability infrastructure, eliminating FL server being a single point of failue. 
+  
+  * Built-in implementations of:
 
-FLARE is about Federated Learning
----------------------------------
-FLARE provides reference Federated Learning algorithms that reflect
-the state of the art (SOTA) for deep learning and machine learning.
+    * Federated Training workflows (scatter-gather, Cyclic);  
+    * Federated Evaluation workflows (global model evaluation, cross site model evalidation);  
+    * Learning algorithms (FedAvg, FedOpt, FedProx) and  
+    * Privacy preserving algorithms (homomorphic encryption, differential privacy) 
+  * Extensible management tools for:
 
-  * Basic Federated Learning Algorithms (FedAvg, FedProx, FedOpt)
-  * Federated Analysis
-  * Deep Learning (CIFAR10, ...)
-  * Personalized Federated Learning (Ditto)
-  * Non-IID (Scaffold)
-  * Medical Applications (MONAI, ...)
-  * Federated XGBoost (coming soon)
+    * Secure provisioning (SSL certificates), 
+    * Orchestration (Admin Console) | (Admin APIs) 
+    * Monitoring of Federated learning experiments. (Aux APIs; Tensorboard visualization) 
+  
+  * A rich set of programmable APIs allowing researchers to create new federated workflows, learning & privacy preserving algorithms. 
 
-With FLARE's pluggable component architecture, you can extend these reference implementations with customized algorithms to fit your needs.
 
-FLARE is a SDK, not a platform
-------------------------------
-We want to enable more people to adopt Federated Learning, whether the user is
+High-level System Architecture
+==============================
+As outlined above, NVIDIA FLARE includes components that allow researchers and developers to build and deploy end-to-end federated learning applications.  The high-level architecture is shown in the diagram below.  This includes the foundational components of the NVIDIA FLARE API and tools for Privacy Preservation and Secure Management of the platform.  On top of this foundation are the building blocks for federated learning applications, with a set of Federation Workflows and Learning Algorithms.
 
-  * a machine learning researcher -- interested in experimenting the latest FL algorithms, or
-  * a data scientist -- interested in applying FL to a real world use case, or
-  * a system integrator -- interested in building a platform to enable Federated Learning for others.
+Alongside this central stack are tools that allow experimentation and proof-of-concept development with the FLARE POC mode and FL Simulator, along with a set of tools used to deploy and manage production workflows.
```

Comment:
If you don't consider POC as one type of FL simulator, then we have to remove FL simulator. As we don't have it yet.

Reply:
I do consider POC the current implementation of the FL simulator, just wasn't sure how to word it.  Maybe better to say "Alongside this central stack are tools that allow experimentation and proof-of-concept development with the FL Simulator (POC mode), along with a set of tools used to deploy and manage production workflows."

Reply:
that looks good.